scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java

```diff
@@ -149,7 +150,7 @@ public void getNextSolution_proceduralGoalCreatesActivities() {
     assertThat(plan.getActivitiesByTime())
         .comparingElementsUsing(equalExceptInName)
         .containsExactlyElementsIn(expectedPlan.getActivitiesByTime());
-    assertEquals(4, problem.getSimulationFacade().countSimulationRestarts());
+    assertEquals(7, problem.getSimulationFacade().countSimulationRestarts());
```

Comment:
Why does this goal now require more simulations? Same is true for other goals in this PR where the number of sims run has increased relative to develop.

Reply:
This has been fixed with the work we did together.